tests/testthat/setup-workflow-ref.R

```diff
@@ -77,7 +78,7 @@ MOD4_ABS_PATH <- file.path(getwd(), LEVEL2_MOD)
 RUN_LOG_ROWS <- 3
 RUN_LOG_COLS <- 8
 CONFIG_COLS <- 4
-SUM_LOG_COLS <- 18
+SUM_LOG_COLS <- 21
```

Comment:
What is the calculation here? This PR appears to add a single new column, the condition number.

Reply:
the two additional heuristics as well. All heuristics get pivoted out to logical columns.

Reply:
Okay, I think I understand. #122 mentions four columns.

- `ofv` was already included, but its meaning has changed (it now excludes the constant).

- `condition_number` is the new column (explicitly) added by this PR.

- ` eta_pval_significant` and `prderr` are the two I'm not sure about. It appears that they are newly included in babylon 2.3.0, and are contained within the summary object field `run_heuristics`. Thus, they are not explicitly mentioned in this PR, and will be pivoted out to new columns unchanged. Is that right?

Reply:
that's right. They are added to `run_heuristics` and all of those are pivoted out to columns.

Reply:
Great, thanks for the explanation.